tests/behat/features/share_link.feature

```diff
@@ -37,6 +37,14 @@ Feature: Share Link
     And the JSON node "data.relationships.shared_node.data.type" should be equal to "node--test"
     And the JSON node "data.relationships.shared_node.data.id" should be equal to "99999999-aaaa-bbbb-ccc-000000000000"
 
+    # Fetch the relationship of the node using include
+    When I send a GET request to "api/v1/share_link/99999999-aaaa-bbbb-ccc-000000000001/999999?include=node"
```

Comment:
Is `node` actually a valid include? Should it include the rubbish one as per the other test?

Reply:
Yeah you can actually pull the node relationship off the share link, since it's a valid relationship in this scenario

Reply:
Have updated the tests to be clearer in this regard